### PR TITLE
[plugin.video.cbc] 4.0.6+matrix.1

### PR DESCRIPTION
**Pull request by micahg**
### Description
Matrix compatible version of the CBC addon.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.cbc/addon.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+	<provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <assets>
+      <icon>icon.png</icon>
+      <fanart>fanart.jpg</fanart>
+    </assets>
+    <summary lang="en">CBC TV Kodi Plugin</summary>
```
Comment (enen92) [inline plugin.video.cbc/addon.xml]: en_GB

File: plugin.video.cbc/addon.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <description lang="en">View online content from the Canadian Broadcasting Corporation.</description>
```
Comment (enen92) [inline plugin.video.cbc/addon.xml]: en_GB

File: plugin.video.cbc/addon.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <platform>all</platform>
+    <language>en</language>
+    <disclaimer lang="en">For bugs, requests or general questions visit the CBC TV thread on the Kodi forum.</disclaimer>
```
Comment (enen92) [inline plugin.video.cbc/addon.xml]: en_GB

File: plugin.video.cbc/addon.xml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <assets>
+      <icon>icon.png</icon>
+      <fanart>fanart.jpg</fanart>
+    </assets>
+    <summary lang="en">CBC TV Kodi Plugin</summary>
+    <description lang="en">View online content from the Canadian Broadcasting Corporation.</description>
+    <platform>all</platform>
+    <language>en</language>
```
Comment (enen92) [inline plugin.video.cbc/addon.xml:22]: Keep this as en

File: plugin.video.cbc/addon.xml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    <disclaimer lang="en">For bugs, requests or general questions visit the CBC TV thread on the Kodi forum.</disclaimer>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```
Comment (enen92) [inline plugin.video.cbc/addon.xml]: Please use a spdx compliant identifier. Should be GPL-2.0-only or GPL-2.0-or-later depending on your interest. See spdx.org

**Review by enen92 — CHANGES_REQUESTED** [review]
Only minors

**Comment by micahg** [comment]
en_GBs should be fixed. Thanks.

**Comment by micahg** [comment]
Do I have to do something to remove that label? Developer has responded.

Thanks,

**Review by enen92 — APPROVED** [review]
All good codewise. I'll ask some american based people if there are any issues we should be aware regarding this particular broadcaster

**Comment by micahg** [comment]
hey @enen92 I'm pretty sure this wont be available in the US -- is there something I should add to it to indicate so?

**Comment by enen92** [comment]
No, appart from language we have no way of filtering add-ons in the UI I'm afraid. Maybe this is something we should keep in mind as a future Kodi feature

**Pull request merged**